odoo/fields.py

```diff
@@ -1294,6 +1294,74 @@ def convert_to_export(self, value, record):
     compare = staticmethod(float_compare)
 
 
+class Uom(Field):
+    """ The decimal precision is taken from the attribute
+
+    :param uom_field: name of the field holding the uom this uom
+                           field is expressed in (default: `uom_id`)
+    """
+    type = 'uom'
```

Comment:
The chosen terminology is nonsensical and misleading: UOM stands for "Unit of Measure".
The value of such a field is not a *unit of measure*, it is a *measure*.
The value of the associated field `uom_field` is the *unit of measure*.

The name of the type should be `measure`.

Reply:
👍 measure is better indeed